.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: install misk-web
+        run: |
+          mkdir ~/.npm-global
+          npm config set prefix '~/.npm-global'
+          PATH=~/.npm-global/bin:$PATH
+          npm install -g @misk/cli
+          miskweb ci-build -e
+      - name: Cache Gradle packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Set gradle version
+        run: echo "version=String.valueOf(project.version).replace(\"-SNAPSHOT\", \"\") + '-$(date +%Y%m%d)-$(git rev-parse --short HEAD)-SNAPSHOT'" > hooks.gradle
```

Comment:
wanna do hours+minutes so if we do 2 releases in a day the versions always go forward?

Reply:
presumably you use `-SNAPSHOT` so there’s no promotion step?

Reply:
re: hours, sure why not

re snapshot: I figured it was the less offensive / easier to convince option, but if you think normal release is fine I can try to get that working. There's a promote target in the latest maven plugin

Reply:
Added hours+mins and it's not pushing snapshots anymore